.markdownlint.json

```diff
@@ -0,0 +1,5 @@
+{
+  "MD013": false,
+  "MD033": false,
+  "MD041": false
```

Comment:
why the difference in codes?

Reply:
`MD013` is line length. It's set to 80 chars per line. That's actually a default setting that is set by VSCode when you create the config using the command pallet.

`MD033` is inline HTML. Many of the Microsoft Templates add `<a name="section-name"></a>` in the headers to make quick jump spots. You can't replicate this behavior in Markdown so this would flag the docs on every header.

`MD041` is the first line should be an `H1` tag. This gets flagged often due to cookiecutter boilerplate.

Reply:
The generated templates don't have the boilerplate code so no need for `MD041` on a project scale. 

`MD033` is kinda controversial so I opted to leave it out on the generated projects and it can also be ignored at the file level with `<!-- markdownlint-disable MD033 -->`

Reply:
got it, makes sense, thnx for explanation!